examples/arithmetics/src/language-server/arithmetics-util.ts

```diff
@@ -4,9 +4,13 @@
  * terms of the MIT License, which is available in the project root.
  ******************************************************************************/
 
-export function applyOp(op: '+' | '-' | '*' | '/'): (x: number, y: number) => number {
-    if (op === '+') return (x, y) => x + y;
-    if (op === '-') return (x, y) => x - y;
-    if (op === '*') return (x, y) => x * y;
-    return (x, y) => y === 0 ? x : x / y;
+export function applyOp(op: '+' | '-' | '*' | '/' | '^' | '%'): (x: number, y: number) => number {
+    switch (op) {
+        case '+': return (x, y) => x + y;
+        case '-': return (x, y) => x - y;
+        case '*': return (x, y) => x * y;
+        case '^': return (x, y) => Math.pow(x, y);
+        case '%': return (x, y) => x % y;
+        default: return (x, y) => y === 0 ? x : x / y;
+    }
```

Comment:
I suggest to explicitly mention the `/` instead of `default`, the type system will then add an error here once further operators are added to the grammar if they are missing here. Applying a division by `default` is quite unconventional 😉.

Reply:
This got added in sometime earlier this week.